iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/VoltageLevelXml.java

```diff
@@ -235,6 +256,25 @@ protected void readSubElements(VoltageLevel vl, NetworkXmlReaderContext context)
                                 NodeBreakerViewInternalConnectionXml.INSTANCE.read(vl, context);
                                 break;
 
+                            case BusXml.ROOT_ELEMENT_NAME:
```

Comment:
We should assert that the minimum version is 1.1, because this is not allowed in 1.0

Reply:
Done.